slither-reports/reports.txt

```diff
@@ -0,0 +1,118 @@
+Reentrancy in ERC721A._mint(address,uint256,bytes,bool) (node_modules/erc721a/contracts/ERC721A.sol#365-407):
+        External calls:
+        - ! _checkContractOnERC721Received(address(0),to,updatedIndex ++,_data) (node_modules/erc721a/contracts/ERC721A.sol#393)
+                - IERC721Receiver(to).onERC721Received(_msgSender(),from,tokenId,_data) (node_modules/erc721a/contracts/ERC721A.sol#570-580)
+        State variables written after the call(s):
+        - _currentIndex = updatedIndex (node_modules/erc721a/contracts/ERC721A.sol#404)
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#reentrancy-vulnerabilities-1
```

Comment:
It is a false positive report from slither.  
For more info: https://github.com/chiru-labs/ERC721A/issues/230#issuecomment-1090334174